encoding/Encoding.java

```diff
@@ -13,10 +13,9 @@
  *
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
- *
  */
 
-package com.vaticle.typedb.core.graph.common;
```

Comment:
straight relocation of the `//graph/common` package to `//encoding`, and split the files into separate packages